src/components/menus/content-contextual-menu.js

```diff
@@ -421,7 +421,7 @@ const ContentContextualMenu = (props) => {
     const getActiveFilterFormId = () => {
```

Comment:
getActiveAutomaticFilterId then ?
We don't need to get ManualFilterId in the content-contextual-menu ?